io/src/main/scala/sbt/io/syntax.scala

```diff
@@ -0,0 +1,30 @@
+package sbt.io
+
+object syntax extends IOSyntax0 {
+  type File = java.io.File
+  type URI = java.net.URI
+  type URL = java.net.URL
+
+  def uri(s: String): URI = new URI(s)
+  def file(s: String): File = new File(s)
+  def url(s: String): URL = new URL(s)
+
+  implicit def fileToRichFile(file: File): RichFile = new RichFile(file)
+  implicit def filesToFinder(cc: Traversable[File]): PathFinder = PathFinder.strict(cc)
+}
+
+import java.io.File
+sealed abstract class IOSyntax0 extends IOSyntax1 {
```

Comment:
this is a complication I hadn't thought about.  We may need to duplicate these hierarchy levels as well in the sbt package object later....